hazelcast/src/main/java/com/hazelcast/config/ExternalDataStoreConfig.java

```diff
@@ -174,4 +189,40 @@ public String toString() {
                 + ", properties=" + properties
                 + '}';
     }
+
+    @Override
+    public void writeData(ObjectDataOutput out) throws IOException {
```

Comment:
nit: for consistency maybe add also setProperty() (btw., I cannot add it on the line with setProperties method xd)

Reply:
fixed: https://github.com/hazelcast/hazelcast/pull/21930/commits/6c01e92e459e0ef1d1f5cfbbb5a477def09f9c31